sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/model/ValueMappingModel.java

```diff
@@ -0,0 +1,47 @@
+package org.sdase.commons.keymgmt.model;
+
+import java.util.*;
+
+@SuppressWarnings("unused")
+public class ValueMappingModel {
+
+  private final Map<String, String> apiToImplBidirectional = new HashMap<>();
+  private final Map<String, String> implToApiBidirectional = new HashMap<>();
+
+  // maps between impl and api with higher precedence than the bidirectional mapping
+  private final Map<String, String> implToApi = new HashMap<>();
+  private final Map<String, String> apiToImpl = new HashMap<>();
+
+  public ValueMappingModel setApiToImplBidirectional(List<ValueMapping> apiToImplBidirectional) {
+    apiToImplBidirectional.forEach(
```

Comment:
Also clear `apiToImplBidirectional` and `implToApiBidirectional`?

Reply:
done thx.